config/urls.py

```diff
@@ -9,6 +9,7 @@
 from rest_framework.routers import DefaultRouter
 from rest_framework.schemas import get_schema_view
 from rest_framework_swagger.renderers import OpenAPIRenderer, SwaggerUIRenderer
+from django_js_reverse import views as django_js_reverse_views
```

Comment:
I think you changed this since last review? Please don't use `as` imports unless there is no other way. It makes tracking annoying since the code calls something that isn't the real name what was imported. Even though PyCharm obviously helps, this is a very strong preference for me.

Always import the exact thing you need, unless there are lots of imports then I think it's ok to import a full module.

Reply:
> I think you changed this since last review?

No, I didn't. This `as` is quite old.

> Please don't use as imports unless there is no other way.

Even if what's imported as a very generic name?